gui/WorldView.cxx

```diff
@@ -1035,13 +1087,40 @@ void WorldView::setMesh(vtkSmartPointer<vtkPolyData> mesh)
 
   // Add this actor to the renderer
   d->renderer->AddActor(d->volumeActor);
-
-  if (mesh->GetPointData()->HasArray("RGB"))
+  bool rgb = false, rgba = false;
+  const char* arrayName = nullptr;
+  const char* rgbName = "RGB";
+  const char* rgbaName = "RGBA";
+  if (mesh->GetPointData()->HasArray(rgbName))
   {
+    rgb = true;
+    arrayName = rgbName;
+  }
+  if (mesh->GetPointData()->HasArray(rgbaName))
+  {
+    rgba = true;
+    arrayName = rgbaName;
+  }
+  if (rgb || rgba)
```

Comment:
```suggestion
  if (arrayName)
```
...and lose `rgb`, `rgba`.

Reply:
While more efficient, it is slightly less clear so I am on the fence here.

Reply:
if we ultimately decide for arrayName, then we probably do not need bool rgb, rgba; unless they get some usage later on; just my 2c

Reply:
Hmm, I might have argued the opposite: "if I found an array to use for coloration" vs. "if I found either RGB or RGBA". Sort of like `if (haveSufficientFunds)` vs `if (bankBalance > costOfItem)`. The latter is "clearer" in the sense of explicitly restating the actual conditions, while the former places more emphasis on the *intent*. I suppose I can see the argument either way (but one is less code 😉).

Reply:
Sounds good.